packages/site/src/components/Navigation/TableOfContents.tsx

```diff
@@ -109,16 +109,15 @@ const Headings = ({ folder, headings, sections }: Props) => {
   );
 };
 
-export function useTocHeight<T extends HTMLElement = HTMLElement>(top?: number) {
+export function useTocHeight<T extends HTMLElement = HTMLElement>(top = 0, inset = 0) {
   const container = useRef<T>(null);
   const toc = useRef<HTMLDivElement>(null);
   const transitionState = useNavigation().state;
   const setHeight = () => {
     if (!container.current || !toc.current) return;
     const height = container.current.offsetHeight - window.scrollY;
     const div = toc.current.firstChild as HTMLDivElement;
-    const MAGIC_PADDING = 16; // I dunno, just go with it ...
-    if (div) div.style.height = `min(calc(100vh - ${top ?? 0}px), ${height + MAGIC_PADDING}px)`;
+    if (div) div.style.height = `min(calc(100vh - ${top}px), ${height + inset}px)`;
```

Comment:
@rowanc1 I think this is a nice solution, lemme know

Reply:
Yeah, this is so much better!! 🚀 

I will test now and get this in and out!